distributions/openhab/src/main/resources/bin/backup

```diff
@@ -79,12 +80,20 @@ setup(){
   CurrentVersion="$(awk '/openhab-distro/{print $3}' "$OPENHAB_USERDATA/etc/version.properties")"
 
   ## Store anything in temporary folders
-  TempDir="/tmp/openhab2/backup"
-  echo "Making Temporary Directory"
-  mkdir -p "$TempDir" || {
+  # Check if we should use TempDir from env $OPENHAB_BACKUPS_TEMP
+  if [ ${OPENHAB_BACKUPS_TEMP} ];  then 
```

Comment:
This would be better as 

```
if [ -n "${OPENHAB_BACKUPS_TEMP}" ]; then
```

Reply:
I changed the patch as you suggested. Do i have to do something else?